packages/editor/src/components/post-author/index.js

```diff
@@ -41,7 +42,7 @@ export class PostAuthor extends Component {
 					className="editor-post-author__select"
 				>
 					{ authors.map( ( author ) => (
-						<option key={ author.id } value={ author.id }>{ author.name }</option>
+						<option key={ author.id } value={ author.id }>{ decodeEntities(author.name) }</option>
```

Comment:
You missed spaces between the parenthesis like this :
```suggestion
						<option key={ author.id } value={ author.id }>{ decodeEntities( author.name ) }</option>
```